app/config.py

```diff
@@ -101,43 +120,36 @@ def get_abs_path(file_path: str):
 # Useful when calling Postfix from an external network
 POSTFIX_SUBMISSION_TLS = "POSTFIX_SUBMISSION_TLS" in os.environ
 
-if "OTHER_ALIAS_DOMAINS" in os.environ:
-    OTHER_ALIAS_DOMAINS = eval(
-        os.environ["OTHER_ALIAS_DOMAINS"]
-    )  # ["domain1.com", "domain2.com"]
-else:
-    OTHER_ALIAS_DOMAINS = []
+OTHER_ALIAS_DOMAINS = parse_environment(
```

Comment:
maybe we can modify `parse_environment` so it accepts the env variable name instead (so `OTHER_ALIAS_DOMAINS` in this case instead of `os.environ.get("OTHER_ALIAS_DOMAINS")` and has a default value when the value isn't there, something like

```python
def parse_environment(env_name: str, default_val):
     if env_name in os.environ: 
          return literal_eval(os.environ[env_name])
     else:
          return default_val
```

Reply:
Thanks, based on your suggestion, I've introduced an additional argument and designed it as a closure instead of fixed value. It will allows us:

- Lazy defined a default value according to our need, ex: a default constant needs other default constant
- Raises when a default constant is not defined, by letting default_factory be None

wdyt?